composites/AlgoliaSearch/SearchBar.js

```diff
@@ -7,43 +7,74 @@ import debounce from "lodash/debounce";
 import colors from "../../style-guide/colors.json";
 import SearchIcon from "../../style-guide/svg/search.svg";
 import { Icon } from "../Plugin/Shared/components/Icon";
+import { YoastButton } from "../Plugin/Shared/components/YoastButton";
+import breakpoints from "../../style-guide/responsive-breakpoints.json";
 
 const messages = defineMessages( {
 	headingText: {
 		id: "searchBar.headingText",
 		defaultMessage: "Search the Yoast knowledge base",
 	},
+	placeholderText: {
+		id: "searchBar.placeholderText",
+		defaultMessage: "Search the knowledge base",
+	},
+	buttonText: {
+		id: "searchBar.buttonText",
+		defaultMessage: "Search",
+	},
 } );
 
-const SearchLabel = styled.label`
-	width: 2em;
-	height: 2em;
-	float: left;
-	margin-top: 0.25em;
-`;
-
 const SearchBarWrapper = styled.div`
-	overflow: hidden;
-	width: 100%;
 	padding: 0 16px;
-	box-sizing: border-box;
+
+	form {
+		display: flex;
+
+		@media screen and (max-width: ${ breakpoints.mobile } ) {
+			flex-wrap: wrap;
+		}
+	}
+
+	@media screen and (max-width: ${ breakpoints.mobile } ) {
+		button {
+			min-width: 100%;
+			margin-top: 1em;
+		}
+	}
+`;
+
+const SearchHeading = styled.h2`
+	font-size: 1em;
+	margin: 0.5em 0 0.5em 58px;
```

Comment:
Not on 8px grid. Does 56px good as well?

Reply:
I know it's not on the 8px grid, but this is the only value that matches the design in the provided screenshot.